app/experimenter/legacy-ui/rapid/components/visualization/HighlightsTable.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import {
+  METRIC,
+  HIGHLIGHTS_METRICS_LIST,
+  BRANCH_COMPARISON,
+  TABLE_LABEL,
+} from "experimenter-rapid/components/visualization/constants/analysis";
+import {
+  METRICS_TIPS,
+  SEGMENT_TIPS,
+} from "experimenter-rapid/components/visualization/constants/tooltips";
+import ResultsRow from "experimenter-rapid/components/visualization/ResultsRow";
+import { ExperimentData } from "experimenter-types/experiment";
+
+const getHighlightMetrics = (
+  featureData: Array<string>,
+): Array<{ value: string; name: string; tooltip: string }> => {
+  const featureNameMappings = featureOptions.reduce((acc, featureRow) => {
+    acc[featureRow.value] = featureRow["name"];
+    return acc;
+  }, {});
+  const highlightMetricsList = [...HIGHLIGHTS_METRICS_LIST];
```

Comment:
It looks like `RESULTS_METRICS_LIST` is already an array, so you don't need to spread it?

Reply:
oh, I was using the spread to make a copy of `HIGHLIGHTS_METRICS_LIST` since it gets modified below and I don't want to modify the original const.

Is there a better way to make a copy or maybe I should leave a comment?

Reply:
I added some comments!

Reply:
Ah ok. That works for me!